tests/AppBundle/Listener/AccessListenerTest.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+namespace Tests\AppBundle\Listener;
+
+use AppBundle\Listener\AccessListener;
+use Symfony\Bundle\FrameworkBundle\Test\KernelTestCase;
+use Symfony\Component\EventDispatcher\EventDispatcher;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\RequestStack;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\Controller\ArgumentResolver;
+use Symfony\Component\HttpKernel\Controller\ControllerResolver;
+use Symfony\Component\HttpKernel\Event\GetResponseEvent;
+use Symfony\Component\HttpKernel\HttpKernel;
+
+class AccessListenerTest extends KernelTestCase
+{
+    /**
+     * @var \Doctrine\ORM\EntityManager
+     */
+    private $em;
+
+    private $accessListener;
```

Comment:
A cleaner way would be to remove the private attribute, create the instance in the first test and return it to the next dependant test.